english/commit-suicide/suicide.md

```diff
@@ -1,2 +1,4 @@
 Click [here](http://www.addictdepression.com/suicide-methods/) for some ideas.
-However, you must consider [these](http://lostallhope.com/things-consider) matters first.
\ No newline at end of file
+However, you must consider [these](http://lostallhope.com/things-consider) matters first.
+
+If you would like someone to talk to about your suicidal inclinations, [here are some hotlines you can call.](https://www.reddit.com/r/SuicideWatch/wiki/hotlines)
```

Comment:
Hm, I'm not sure why Casey thinks this link is broken. If you break up the long line, I'll be happy to merge this.